.github/workflows/pr-code-scan.yml

```diff
@@ -66,3 +66,15 @@ jobs:
         with:
           name: ${{ matrix.job_name }}
           path: ${{ github.workspace }}/.github/workflows/scripts/codeScan/${{ matrix.job_name }}.*
+
+  shell-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout out Repo
+        uses: actions/checkout@v4
+
+      - name: ShellCheck
+        uses: ludeeus/action-shellcheck@2.0.0
```

Comment:
There seems to be quite a few possible action alternatives for shellcheck.  I have no clue which one of them would be best, but was there any particular reason for selecting this one instead of e.g: https://github.com/marketplace/actions/differential-shellcheck ?

(Which would check only changes done within the PR, instead of forcing submitter fixing also issues shelllcheck reports for already existing code.)